src/Components/Server/src/CircuitOptions.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNetCore.Authorization;
```

Comment:
This line probably isn't required on its own.

Reply:
Nope, this is cleanup.